installation/cmd/run.sh

```diff
@@ -23,7 +23,8 @@ fi
 kyma provision minikube
 kyma install -c $INSTALLER_CR_PATH -o $OVERRIDES_KYMA --source $KYMA_SOURCE
 
-bash "${ROOT_PATH}"/installation/scripts/run-compass-installer.sh
```

Comment:
So if no compass will be installed also the line 32 should be modified, those no longer will be available: `adapter-gateway.kyma.local adapter-gateway-mtls.kyma.local compass-gateway-mtls.kyma.local compass-gateway-auth-oauth.kyma.local compass-gateway.kyma.local compass.kyma.local compass-mf.kyma.local`